unitary/alpha/quantum_world.py

```diff
@@ -484,6 +486,35 @@ def get_histogram(
                 histogram[idx][cast(int, result[idx])] += 1
         return histogram
 
+    def get_histogram_with_all_objects_as_key(
```

Comment:
I think we might want to come up with a better name.  Maybe get_correlated_histogram?  @cognigami @cantwellc any suggestions?

Reply:
Thanks Doug. It's updated. @cognigami @cantwellc please let me know if you have other suggestions. Thanks.